providers/certification-client/units/client-cert-iot-ubuntucore-24.pxu

```diff
@@ -35,25 +35,25 @@ bootstrap_include:
   interface
   net_if_management
 nested_part:
-  submission-cert-full
-  self-manual
-  ubuntucore-manual
   audio-manual
-  bluetooth-manual
+  bluetooth-cert-manual
   camera-manual
   cpu-manual
   disk-manual
   ethernet-manual
+  led-indicator-manual
   led-manual
   mediacard-manual
   memory-manual
-  monitor-manual
```

Comment:
Why we don't need monitor manual test for Ubuntu Core? Some of the IoT devices do have display support such as HDMI and DP.